java/core/src/java/org/apache/orc/impl/RecordReaderUtils.java

```diff
@@ -770,18 +770,18 @@ void readRanges(FSDataInputStream file, boolean allocateDirect, double extraByte
     }
 
     static ChunkReader create(BufferChunk from, BufferChunk to) {
-      long f = Integer.MAX_VALUE;
-      long e = Integer.MIN_VALUE;
+      long f = Long.MAX_VALUE;
```

Comment:
Recommendation: This is already existing so please feel free to ignore. 
Since we are touching the code, Can we rename the variables for easy reading?
Couldn't understand the meaning of  f, e, cf, ce 😢

Reply:
Great, let me try.

Reply:
Actually, I'm opposite to @mystic-lama 's comment.

We don't want to mix a bug fix and a style fix (or refactorying) in the same PR. It's because it increases the complexity indeed.

If we want to rename this, it should be done before or after this PR.

Please don't change the existing variable name in this PR.

Reply:
Ack, @dongjoon-hyun has a point. @yebukong we can do that in follow up PR or may be leave it as is :)
Apologies for the noise.